.github/workflows/lint.yml

```diff
@@ -28,7 +27,6 @@ jobs:
         uses: ocaml/setup-ocaml@v3
         with:
           ocaml-compiler: "5.2"
-          dune-cache: true
```

Comment:
You still haven't explained this change

Reply:
From dune 3.17, dune cache no longer works with opam workflow. There is a work going on upstream about this, but there is no timeline yet...